webapp/src/components/gallery/galleryCard.tsx

```diff
@@ -92,8 +92,8 @@ const GalleryCard = React.memo((props: Props) => {
                             id='duplicate'
                             name={intl.formatMessage({id: 'GalleryCard.duplicate', defaultMessage: 'Duplicate'})}
                             onClick={() => {
-                                TelemetryClient.trackEvent(TelemetryCategory, TelemetryActions.DuplicateCard, {board: board.id, card: card.id})
-                                mutator.duplicateCard(card.id)
+                                TelemetryClient.trackEvent(TelemetryCategory, TelemetryActions.DuplicateCard, {board: props.board.id, card: card.id})
```

Comment:
Was this change intentional? 'props' was just removed here. https://github.com/mattermost/focalboard/pull/1739/files

Reply:
Not intentional. Never had intention to change that. Thanks for pointing out. I've reverted.